packages/core/src/modules/oob/OutOfBandApi.ts

```diff
@@ -430,12 +465,13 @@ export class OutOfBandApi {
 
     const { outOfBandInvitation } = outOfBandRecord
     const { label, alias, imageUrl, autoAcceptConnection, reuseConnection, routing } = config
-    const { handshakeProtocols } = outOfBandInvitation
-    const services = outOfBandInvitation.getServices()
-    const messages = outOfBandInvitation.getRequests()
+    const services = outOfBandInvitation?.getServices()
+    const messages = outOfBandInvitation?.getRequests()
     const timeoutMs = config.timeoutMs ?? 20000
 
-    const existingConnection = await this.findExistingConnection(outOfBandInvitation)
+    const { handshakeProtocols } = outOfBandInvitation
+
+    const existingConnection = outOfBandInvitation ? await this.findExistingConnection(outOfBandInvitation) : undefined
```

Comment:
In which case is outOfBandInvitation undefined? It looks as if you an destructure it above,

Reply:
At the beginning, I was going to make outOfBandInvitation optional, but later on I switched to this more conservative approach to not break so many things... and forgot to revert this line.